backend/src/db/migrations/20230404102938_add_project_type_in_projects_table.ts

```diff
@@ -0,0 +1,21 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'projects';
+
+const ColumnName = {
+  TYPE: 'type',
+};
+
+function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(TABLE_NAME, (table) => {
+    table.string(ColumnName.TYPE).notNullable();
```

Comment:
If values for this col are stale - maybe it would be better to make it enum,, and also `type` is a preserved word for postgres - won't it be better to name it with something else - like `category` etc

Reply:
enum for sure